### PR TITLE
Merge release/lavida (4.5.0) branch into master

### PR DESCRIPTION
**Pull request by paulbourelly999**
# PR Details
## Description
**Requires #18  closes**

## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING**](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/Contributing.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
@MishkaMN

### CODE REVIEW

File: .github/workflows/dockerhub.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: Docker Hub build
+on:
+  push:
+    branches:
+      - "develop"
+      - "master"
+      - "release/*"
+      - "release/lavida"
```
Comment (paulbourelly999) [inline .github/workflows/dockerhub.yml:8]: I need to remove this.

**Comment by MishkaMN** [comment]
I think it's pretty low chance that this is happening but should we make this point to previous release 4.4.3? 
https://github.com/usdot-fhwa-stol/carla-sensor-lib/blob/release/lavida/Dockerfile#L7
Just in case if the egg file gets removed or moved on develop?
@paulbourelly999

File: .github/workflows/dockerhub.yml (hunk 1 above)
Comment (MishkaMN) [reply .github/workflows/dockerhub.yml:8]: yeah let's also not forget to add the trigger for carma-system-*

**Review by MishkaMN — CHANGES_REQUESTED** [review]

**Comment by paulbourelly999** [comment]
> I think it's pretty low chance that this is happening but should we make this point to previous release 4.4.3 or current one (after we tag carma-carla-integration) although it hasn't existed yet 4.5.0? https://github.com/usdot-fhwa-stol/carla-sensor-lib/blob/release/lavida/Dockerfile#L7 Just in case if the egg file gets removed or moved on develop? @paulbourelly999
> 
> I am okay either way

I say we just leave it. this needs to be fixed to install CARLA from release version anyway. Addressed other issues in new PR #18

**Review by MishkaMN — APPROVED** [review]

**Pull request merged**